### PR TITLE
component(onecarouseltwostories): added one carousel two stories one …

### PR DESCRIPTION
**Pull request by supreeth-spiger**
added one carousel two stories one  ad one widget component

### CODE REVIEW

File: src/components/atoms/SlideBigStoryCard/stories.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    const newProps = produce(story, draft => {
+      draft.story.url = null;
+    });
+    return <SlideBigStoryCard {...newProps} />;
+  })
+  .add("long headline", () => {
+    const newProps = produce(story, draft => {
+      draft.story.alternative = null;
+      draft.story.headline =
+        "கமலே கவலைப்படும் அளவுக்கு இருக்கிறதா கமலே கவலைப்படும் அளவுக்கு இருக்கிறதா கமலே கவலைப்படும் அளவுக்கு இருக்கிறதா";
```
Comment (ajaymenon0) [inline src/components/atoms/SlideBigStoryCard/stories.js]: Replace

File: src/components/atoms/SmallStoryDesktopBigNoBorder/index.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { get } from "lodash";
```
Comment (ajaymenon0) [inline src/components/atoms/SmallStoryDesktopBigNoBorder/index.js]: from `lodash/get`

File: src/components/rows/OneCarouselTwoStoriesOneAdOneWidget/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { get } from "lodash";
+import Carousel from "../../atoms/Carousel";
+import SmallStoryCardDesktopBigNoBorder from "../../atoms/SmallStoryDesktopBigNoBorder";
+import "./oneCarouselTwoStoriesOneAdOneWidget.m.css";
+import SlideBigStoryCard from "../../atoms/SlideBigStoryCard";
+import { DfpAd } from "../../atoms/DfpAd";
```
Comment (ajaymenon0) [inline src/components/rows/OneCarouselTwoStoriesOneAdOneWidget/index.js]: We can remove dfpAd component.

File: src/components/utils/story.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import get from "lodash/get";
```
Comment (ajaymenon0) [inline src/components/utils/story.js]: Any export from this file doesn't seem to be used anywhere

File: src/components/atoms/DfpAd/index.js [hunk 5]
```diff
@@ -0,0 +1,262 @@
+import React from "react";
```
Comment (ajaymenon0) [inline src/components/atoms/DfpAd/index.js]: Remove this file

**Review by ajaymenon0 — CHANGES_REQUESTED** [review]

File: src/components/atoms/SlideBigStoryCard/stories.js (hunk 1 above)
Comment (supreeth-spiger) [reply src/components/atoms/SlideBigStoryCard/stories.js]: done

File: src/components/atoms/SmallStoryDesktopBigNoBorder/index.js (hunk 2 above)
Comment (supreeth-spiger) [reply src/components/atoms/SmallStoryDesktopBigNoBorder/index.js]: done

File: src/components/utils/story.js (hunk 4 above)
Comment (supreeth-spiger) [reply src/components/utils/story.js]: not being imported any where so removed the file

File: src/components/rows/OneCarouselTwoStoriesOneAdOneWidget/index.js (hunk 3 above)
Comment (supreeth-spiger) [reply src/components/rows/OneCarouselTwoStoriesOneAdOneWidget/index.js]: removed dfp ad component

File: src/components/atoms/DfpAd/index.js (hunk 5 above)
Comment (supreeth-spiger) [reply src/components/atoms/DfpAd/index.js]: removed

File: src/components/atoms/ResponsiveImageWithFallback/index.js [hunk 6]
```diff
@@ -2,10 +2,9 @@ import React from "react";
 import PropTypes from "prop-types";
 import { ResponsiveImage, ResponsiveSource } from "@quintype/components";
 import VikatanStoryFallback from "../../shared/icons/fallbackImage";
-import { PremiumBadge } from "../PremiumBadge";
 import { omit } from "lodash";
```
Comment (nandap4790) [inline src/components/atoms/ResponsiveImageWithFallback/index.js]: `lodash/omit`

File: src/components/atoms/SlideBigStoryCard/index.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+        <div styleName="text-wrapper">
+          {contributor && (
+            <Contributor
+              name={contributor["name"]}
+              type={contributorRole}
+              iconColor="#4a4a4a"
+              className="contributor"
+            />
+          )}
+          <h3 styleName={`${"headline"} ${hasTruncatedHeadline ? "truncated" : ""} `}>{storyData.headline}</h3>
```
Comment (nandap4790) [inline src/components/atoms/SlideBigStoryCard/index.js]: Use headline component

File: src/components/atoms/SmallStoryDesktopBigNoBorder/index.js [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+        sources={generateImageSources(
+          { aspectRatio: [4, 3], screenWidthCoverage: 0.25 },
+          { aspectRatio: [2, 1], screenWidthCoverage: 0.34 }
+        )}
+      />
+      <div styleName="text-wrapper">
+        {contributor && (
+          <Contributor name={contributor["name"]} type={contributorRole} iconColor="#4a4a4a" className="contributor" />
+        )}
+        <h3 styleName={`${hasTruncatedHeadline ? "truncated" : ""} ${"headline"}`}>{storyData.headline}</h3>
```
Comment (nandap4790) [inline src/components/atoms/SmallStoryDesktopBigNoBorder/index.js:50]: Headline

File: src/components/utils/section.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+import { get } from "lodash";
```
Comment (nandap4790) [inline src/components/utils/section.js]: lodash/get

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: src/components/atoms/ResponsiveImageWithFallback/index.js (hunk 6 above)
Comment (supreeth-spiger) [reply src/components/atoms/ResponsiveImageWithFallback/index.js]: done

File: src/components/atoms/SmallStoryDesktopBigNoBorder/index.js (hunk 8 above)
Comment (supreeth-spiger) [reply src/components/atoms/SmallStoryDesktopBigNoBorder/index.js:50]: changes made

File: src/components/utils/section.js (hunk 9 above)
Comment (supreeth-spiger) [reply src/components/utils/section.js]: done

**Pull request merged**